firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionTesterApiClient.java

```diff
@@ -42,6 +42,8 @@
   private static final String INSTALLATION_AUTH_HEADER = "X-Goog-Firebase-Installations-Auth";
   private static final String X_ANDROID_PACKAGE_HEADER_KEY = "X-Android-Package";
   private static final String X_ANDROID_CERT_HEADER_KEY = "X-Android-Cert";
+  private static final String X_CLIENT_VERSION_HEADER_KEY = "X-Client-Version";
+  // Format of "X-Client-Version": "{ClientId}/{ClientVersion}"
```

Comment:
I feel like this comment should be above the declaration?

Reply:
Done :)